backend/benefit/locale/fi/LC_MESSAGES/django.po

```diff
@@ -1496,6 +1601,26 @@ msgstr "palveluehtojen hyväksyntä"
 msgid "terms of service approvals"
 msgstr "palveluehtojen hyväksynnät"
 
+#, fuzzy
+#~| msgid "Decision date"
```

Comment:
@rikuke's XML translations?

Reply:
@EmiliaMakelaVincit did you do it the updated way? Or is there something wrong with this again 🤔 

https://github.com/City-of-Helsinki/yjdh/tree/main/backend/benefit#updating-translations

Reply:
Yeah, I used the new command with the extension argument.

I think in this case it's working as intended 🤔 The default Ahjo strings were changed a bit but were not updated before in the translation files – the only location the string "type of the decision proposal template section" exists in the repository seems to be the .po files themselves and a superseded migration. They should've probably been cleaned up earlier already, but they weren't, and now that the string has disappeared, the command moved the (already previously fuzzy) translations into the deleted translations section of the file.

The additions on lines 533-593 seem to add the equivalent current translation keys in.